src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -77,6 +89,12 @@ public void disengageRatchet() {
         rightServo.setPosition(0);
         ratchetEngaged = false;
     }
+    public double getEncoderPosition() {
+        return climberLeft.getEncoder().getPosition();
+    }
+    public boolean isExtendedLimitSwitchNotPressed() {
```

Comment:
Can you add a isExtendedLimitSwitchPressed instead aswell as one for Left and Right seperately, though the latter my not be needed nor wanted.